apps/onlyoffice_docker.sh

```diff
@@ -160,7 +160,7 @@ fi
 # OnlyOffice URL (onlyoffice.sh)
 SUBDOMAIN=$(input_box "OnlyOffice subdomain e.g: office.yourdomain.com\n\nNOTE: This domain must be different than your Nextcloud domain. They can however be hosted on the same server, but would require seperate DNS entries.")
 # Nextcloud Main Domain (onlyoffice.sh)
-NCDOMAIN=$(input_box "Nextcloud domain, make sure it looks like this: cloud\\.yourdomain\\.com")
+NCDOMAIN=$(occ_command_no_check config:system:get overwrite.cli.url | sed 's#https://##;s#/##')
```

Comment:
What if that value isn't set? Better to hardcode it if you ask me.

Reply:
The overwrite.cli.url must be set, else Nextcloud doesn't work correctly. So I am sure that this works.

Reply:
If the user haven't run the LE script the domain is 'localhost'. Have you tested that as well so that it doesn't bug out if nothing is set to something not being a domain?

Reply:
No, I haven't tested this detail. But my testing in the past has showed that the curl command always fails if self singed certificates are used which is the case here.

Reply:
I've tested this just now and can confirm my previous comment